From: Marcos Paulo de Souza <marcos.souza.org@gmail.com>
Date: Tue, 3 Sep 2019 08:04:02 -0600
Subject: [PATCH] Documenation: switching-sched: Remove notes about elevator
 argument
References: bsc#1156419,jsc#SLE-8281
Git-commit: f97eeb6cfd527f5d40167bf20359eee3c1d23a6d
Patch-mainline: v5.4-rc1

This argument was ignored since blk-mq was set as default, so remove it
from documentation.

Reviewed-by: Hannes Reinecke <hare@suse.com>
Reviewed-by: Christoph Hellwig <hch@lst.de>
Signed-off-by: Marcos Paulo de Souza <marcos.souza.org@gmail.com>

.txt file is now .rst

Signed-off-by: Jens Axboe <axboe@kernel.dk>
Signed-off-by: Hannes Reinecke <hare@suse.de>
---
 Documentation/block/switching-sched.rst | 4 ----
 1 file changed, 4 deletions(-)

diff --git a/Documentation/block/switching-sched.rst b/Documentation/block/switching-sched.rst
index 42042417380e..520f6b857544 100644
--- a/Documentation/block/switching-sched.rst
+++ b/Documentation/block/switching-sched.rst
@@ -2,10 +2,6 @@
 Switching Scheduler
 ===================
 
-To choose IO schedulers at boot time, use the argument 'elevator=deadline'.
-'noop' and 'cfq' (the default) are also available. IO schedulers are assigned
-globally at boot time only presently.
-
 Each io queue has a set of io scheduler tunables associated with it. These
 tunables control how the io scheduler works. You can find these entries
 in::
-- 
2.16.4

